src/components/components/ReinforcementStatement/ReinforcementStatement.scss

```diff
@@ -0,0 +1,55 @@
+.Reinforcement {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin: 96px;
+  gap: 96px;
+
+
+  &__image {
+    border-radius: 8px;
+    max-width: 50%;
+  
+}
+  &__statement {
+    display:flex;
+    flex-direction: column;
+    max-width: 50%;
+
+    .Reinforcement__header {
+      color: var(--color-bunting);
+      font-weight: 700;
+      font-size: 32px;
```

Comment:
i dont think you need the font-size here as we have it globally in the index.scss

Reply:
Removed